app/views/pages/live.html.haml

```diff
@@ -0,0 +1,77 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'live.css'
+
+= render "layouts/hackathon_manager/flashes"
+= render "layouts/nav"
+
+.section#timer
+  .timer__section
+    %h2.text-center Timer
+    .timer__column__container
+      .timer__left
+        %h4.text-center Timer #1
+        .timer__container
+          %p.text-center{:id => "timer_countdown"}
+          :javascript
+            var x = setInterval(function() {
```

Comment:
Since this is a globally-accessible variable (global since it's inside a bare `<script>` body), using a variable name other  `x` would be safer. If another one of our scripts (or a poorly-implemented 3rd party library) also used the global variable `x`, we'd hit some problems.